var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -97,31 +96,41 @@ class Llvm(CMakePackage, CudaPackage):
         default=False,
         description="Include debugging code in OpenMP runtime libraries",
     )
-    variant("lldb", default=True, when="+clang", description="Build the LLVM debugger")
```

Comment:
Why is this being removed?

Reply:
This is also part of [2d509b5](https://github.com/spack/spack/pull/32476/commits/2d509b51465d4132610eebea30984524096b7d10) (`llvm: allow for explicit 'llvm~clang~flang~lldb~z3'`)

Reply:
Makes sense, moreso since I found the conflicts, a small refactor would be nice but I agree with the result. Thanks for working through that.